local-docker-compose.yml

```diff
@@ -47,24 +47,56 @@ services:
       - "8888:8888"
     user: ${MATLAB_UID}:${MATLAB_GID}
     working_dir: /home/muser/notebooks
-    command: > 
-      /bin/bash -c "
-        ## Remove mym
-        rm -R /src/mym;
-        ## Set dj path
-        matlab -nodisplay -r \"\
-            addpath('/src');\
-            savepath;\
-          \";
-        ## Interactive Jupyter Notebook environment
-        jupyter notebook;
-      "
+    command:
+      - /bin/bash
+      - -c
+      - |
+        ORIG_DIR=$$(pwd)
+        mkdir ~/Documents
+        cd /src
+        # package into toolbox, and install
+        matlab -nodisplay -r "
```

Comment:
no strong opinions but this script may be getting a little long for inline, might make sense to break out. YMMV

Reply:
Thought about this before but ultimately, this file is committed merely as reference for local development. It really does not have a real place in the source. Justified it by keeping it as a single file, however, if we split this into more, we might need to move to a subdirectory or in docs so it doesn't clutter or create additional confusion when others review actual source.